docs/specification/draft/basic/authorization.mdx

```diff
@@ -302,6 +345,68 @@ Servers **MUST** return appropriate HTTP status codes for authorization errors:
 | 403         | Forbidden    | Invalid scopes or insufficient permissions |
 | 400         | Bad Request  | Malformed authorization request            |
 
+#### Scope Error Handling
+
+This section covers handling insufficient scope errors during runtime operations when
+a client already has a token but needs additional permissions. This follows the error
+handling patterns defined in [OAuth 2.1 Section 5](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-13#section-5)
+and leverages the metadata fields from [RFC 9728 (OAuth 2.0 Protected Resource Metadata)](https://datatracker.ietf.org/doc/html/rfc9728).
+
+##### Runtime Insufficient Scope Errors
+
+When a client makes a request with an access token with insufficient
+scope during runtime operations, the server **SHOULD** respond with:
+
+- `HTTP 403 Forbidden` status code (per [RFC 6750 Section 3.1](https://datatracker.ietf.org/doc/html/rfc6750#section-3.1))
+- `WWW-Authenticate` header with the `Bearer` scheme and additional parameters:
+  - `error="insufficient_scope"` - indicating the specific type of authorization failure
+  - `scope="required_scope1 required_scope2"` - specifying the minimum scopes needed for the operation
+  - `resource_metadata` - the URI of the Protected Resource Metadata document (for consistency with 401 responses)
+  - `error_description` (optional) - human-readable description of the error
+
+**Server Scope Management**: When responding with insufficient scope errors, servers
+**SHOULD** include the scopes needed to satisfy the current request in the `scope`
+parameter. Servers have flexibility in determining which scopes to include:
+
+- **Minimum approach**: Include only the newly required scopes for the specific operation
```

Comment:
I think this is slightly inconsistent with this line below:
> (trusting the server to include any necessary existing scopes along with newly required scopes)

Maybe we just delete this line?

Reply:
@pcarleton Yes, I would love to provide an opinionated minimum here. In my experience there is a _lot_ of variability in how Authorization Servers treat existing scopes - the AS will know what its _own_ policy around existing scopes will be, but we obviously can't expect the client to know that policy.

Perhaps:
```suggestion
- **Minimum approach**: Include the newly-required scopes for the specific operation. Include any existing granted scopes as well, if they are required prevent clients from losing previously granted permissions.
```